README.md

```diff
@@ -12,13 +10,108 @@ Simply run:
 
 $ inspec plugin install train-habitat
 
+## Using the plugin as a Train Transport
+
+### Dual-mode transport
+
+Because habitat exposes some facts by its HTTP Gateway API, and some facts by its CLI tool `hab`, this Train Transport has three modes of operation:
+
+ * Using only the HTTP API (no ability to query packages, but rich ability to query rings)
+ * Using only the `hab` CLI command (limitations TBD)
+ * Using both (full capabilities)
+
+When creating a train-habitat Connection, there are thus two sets of options, prefixed with `api_` and `cli_` respectively. You must provide at least one set.
+
+### API-Mode options
+
+#### api_url
+
+An HTTP or HTTPS URL which identifies a Supervisor HTTP Gateway.  If the port is omitted from the URL, the API standard port of 9631 is assumed; to use port 80, specify it explicitly.
+
+#### api_auth_token
+
+The supervisor may be configured to require [Bearer Token Authorization](https://www.habitat.sh/docs/using-habitat/#monitor-services-through-the-http-api), in which the client and the gateway use a pre-shared secret. Specify the secret here to use this feature.
```

Comment:
Where is "here" when specifying the secret? (last sentence)

Reply:
How about 'Use this option to specify the secret'?

Reply:
Prefect.